src/screenComponents/missileTubeControls.cpp

```diff
@@ -141,6 +141,11 @@ void GuiMissileTubeControls::onDraw(sf::RenderTarget& window){
             rows[n].fire_button->setText(tr("missile","Firing"));
             rows[n].loading_bar->hide();
         }
+
+        if (my_spaceship->current_warp > 0.0)
+        {
+            rows[n].fire_button->disable()->show();
```

Comment:
Shouldn't be calling "show()" here, that will make the button show up even if the tube is loading a missile during warp